src/Microsoft.ML.Transforms/Text/LdaTransform.cs

```diff
@@ -693,25 +693,10 @@ internal static LatentDirichletAllocationTransformer TrainLdaTransformer(IHostEn
             return new LatentDirichletAllocationTransformer(env, ldas, columnMappings, columns);
         }
 
-        private void Dispose(bool disposing)
-        {
-            if (_ldas != null)
-            {
-                foreach (var state in _ldas)
-                    state?.Dispose();
-            }
-            if (disposing)
-                GC.SuppressFinalize(this);
-        }
-
         public void Dispose()
         {
-            Dispose(true);
-        }
-
-        ~LatentDirichletAllocationTransformer()
```

Comment:
📝 This finalizer was unnecessary. The type is sealed and does not directly own any unmanaged resources.

Reply:
I'm leaving this change for a separate pull request.